### DISCUSSION TITLE
Error: Database Initialization Error

### DISCUSSION THREAD

**Discussion by oshybrid** [requarks/wiki#7381] [Help / Questions]
Sorry not a DB guy.  I see what the error is saying but I don't remember seeing any instructions saying i should do what the error suggest.  Did i miss something?...PS, can't wait for 3.0

> PS C:\wiki> node server
> Loading configuration from C:\wiki\config.yml... OK
> 2024-10-11T14:04:56.312Z [MASTER] info: =======================================
> 2024-10-11T14:04:56.314Z [MASTER] info: = Wiki.js 2.5.304 =====================
> 2024-10-11T14:04:56.314Z [MASTER] info: =======================================
> 2024-10-11T14:04:56.314Z [MASTER] info: Initializing...
> 2024-10-11T14:04:57.411Z [MASTER] info: Using database driver pg for postgres [ OK ]
> 2024-10-11T14:04:57.417Z [MASTER] info: Connecting to database...
> 2024-10-11T14:04:57.657Z [MASTER] info: Database Connection Successful [ OK ]
> 2024-10-11T14:04:57.716Z [MASTER] error: Database Initialization Error: create table "migrations" ("id" serial primary key, "name" varchar(255), "batch" integer, "migration_time" timestamptz) - permission denied for schema public

**Comment by NGPixel** [comment]
Check your database user permissions.
